be/tool/beck.c

```diff
@@ -459,6 +464,15 @@ static bool  dry_run = false;
 static bool  disable_directio = false;
 static bool  signaled = false;
 
+static struct m0_be_tx_bulk_cfg default_tb_cfg = (struct m0_be_tx_bulk_cfg){
+		.tbc_q_cfg = {
+			.bqc_q_size_max       = 1000,
+			.bqc_producers_nr_max = 1,
+		},
+			.tbc_workers_nr       = 0x40,
```

Comment:
why are these constants used?
Do we have some explanation why they are?

Reply:
Added explanation